README.md

```diff
@@ -350,6 +350,29 @@ limit the commands to the specific dataset!_
    1. Ask AWS support to remove the delete markers returned by
       `aws s3api list-object-versions --bucket=linz-geostore --prefix=DATASET_TITLE/ | jq .DeleteMarkers`.
 
+## Geostore Release
+
+### Versioning and Frequency
+
+We aim to release at the end of each agile sprint (fortnightly), or whenever required (e.g. bugfix,
+feature rollout).
+
+Geostore follows [semantic versioning](http://semver.org/). Each release is incrementally tag with
+`release-x.yy.z` (where x.yy.z denotes a version number).
```

Comment:
As per discussion, maybe something like "`release-major.minor.patch` (for example, `release-12.10.2`)"?